codecs/h264_packet.go

```diff
@@ -33,6 +34,48 @@ const (
 
 func annexbNALUStartCode() []byte { return []byte{0x00, 0x00, 0x00, 0x01} }
 
+// returns true if nals byte stream starts with subtitle sei and the it's position end index
```

Comment:
`// returns true if nals byte stream starts with subtitle sei and its position end index`